app/src/main/java/amazingme/activities/app/LoginActivity.java

```diff
@@ -13,7 +13,6 @@
 import com.google.firebase.auth.FirebaseAuthInvalidUserException;
```

Comment:
same here. this page shouldn't have changed as a result of your changes. 

revert this file as well.